config/application.rb

```diff
@@ -29,6 +29,8 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
 
+    config.admin_group = ENV.fetch('ADMIN_GROUP', 'cn=umg/up.libraries.scholarsphere.service-team,dc=psu,dc=edu')
```

Comment:
this should be `cn=umg/up.dlt.scholarsphere-admin,dc=psu,dc=edu`

Reply:
fixed!